tests/load.rs

```diff
@@ -0,0 +1,21 @@
+use sbp_settings::{settings, Setting, SettingKind};
```

Comment:
Have to use separate crates for testing `load`/`load_from_path` because once you load the settings they are fixed for the rest of the tests